src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
```

Comment:
Can we follow the same sections' ordder than other releases? I think it is:
- Features
- Improvements
- Bugfixes